docs/src/architecture/accounts.md

```diff
@@ -80,3 +76,10 @@ The process is as follows:
 * Bob executes a transaction and creates a note that contains an asset for Alice
 * Alice consumes Bob's note to receive the asset in a transaction
 * Depending on the account storage mode (private vs. public) and transaction type (local vs. network) the Operator receives new Account ID eventually and - if transaction is correct - adds the ID to the Account DB
+
+## Account storage modes
+Account data - stored by the Miden Node - can be public, private, or encrypted. The third and fourth most significant bits of the account ID specifies whether the account data is public `00`, encrypted `01`, or private `11`.
```

Comment:
Note that encrypted state ins't implemented currently.

Reply:
Indeed, Bobbin wanted me to include it already